src/sphinx_2i2c_theme/__init__.py

```diff
@@ -22,7 +22,7 @@ def update_config(app):
         app.config.ogp_site_url = "2i2c.org"
 
     # Social previews config
-    social_cards = app.config.__dict__.get("ogp_social_cards", {})
+    social_cards = app.config.__dict__.get("ogp_social_cards") or {}
```

Comment:
Previously, social_cards could be assigned `None` if `ogp_social_cards` had the value None explicitly set, as compared to not being a key in the dictionary.